lib/livebook_web/live/session_live/secrets_component.ex

```diff
@@ -53,16 +56,17 @@ defmodule LivebookWeb.SessionLive.SecretsComponent do
               <.secret_with_badge
                 :for={{secret_name, _} <- Enum.sort(@secrets)}
                 secret_name={secret_name}
-                secret_origin={:session}
+                secret_hub_id={:session}
```

Comment:
I would keep it as origin only because it isn't an actual hub id and we don't need to pass the hub id anyway.

```suggestion
                secret_origin={:session}
```